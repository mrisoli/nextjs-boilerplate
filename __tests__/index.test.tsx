import { shallow } from 'enzyme';
import * as React from 'react';
import renderer from 'react-test-renderer';

import App from '../pages/index';

describe('With React Testing Library', () => {
  it('Shows welcome message', () => {
    const app = shallow(<App />)

    expect(app.find('div').text()).toEqual('Welcome to next.js!');
  })
})

describe('With Snapshot testing', () => {
  it('Should match Snapshot', () => {
    const component = renderer.create(<App />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  })
})
