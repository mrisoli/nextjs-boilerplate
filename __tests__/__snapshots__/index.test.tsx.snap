// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot testing Should match Snapshot 1`] = `
<div>
  Welcome to next.js!
</div>
`;
